1-exercises/D-do-while/exercise.js

```diff
@@ -7,7 +7,13 @@
 */
 
 function evenNumbersSum(n) {
-    // TODO
+    let i=0
+    let sumOfEven=0;
+    do{
+        sumOfEven=sumOfEven+i*2;
+        i++;
+    }while(i<n)
+    return sumOfEven;
```

Comment:
Nice codes. You can also do it this way 
 let sum= 0;
    let i = 0;
    do {
        sum += i * 2;
         i = i + 1;
    } while (i < n);
    return sum;